### PR TITLE
chore: remove feature flag for ramps

### LINKED ISSUE #26443: [Task]: Remove feature flag from CE repo

### PR DESCRIPTION
**Pull request by ayushpahwa**
## Description
PR to show ramps to everyone on CE, removing feature flag.

#### PR fixes following issue(s)
Fixes #26443

#### Type of change

- New feature (non-breaking change which adds functionality)

- [x] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Test Plan
1. Give a non appsmith user Administrator rights - [adminuser1@app.in](mailto:adminuser1@app.in) - administrator - able to login with this user and see ramps for canvas/preview/deploy modes, on Edit DS page and on Queries page
2. [devuser1@app.in](mailto:devuser1@app.in) - developer - able to login with this user and see ramps for canvas/preview/deploy modes, on Edit DS page and on Queries page
3. [viewuser1@app.in](mailto:viewuser1@app.in) - appviewer - able to login with this user - and only launch the app - where no ramps are visible and no switch env dropdown
4. Make application public - no ramps should be visible - and no switch dropdown should be present
5. Gave above permissions to appsmith user as well and tried if same behavior. Works fine. 
6. Detailed cases - https://github.com/appsmithorg/TestSmith/issues/2489
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [x] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by ankitsrivas14 — DISMISSED** [review]
LGTM

File: app/client/src/ce/components/SwitchEnvironment/index.tsx [hunk 1]
```diff
@@ -94,7 +92,7 @@ export default function SwitchEnvironment({}: Props) {
   //this parameter helps us to differentiate between the two.
   const isDatasourceViewMode = useSelector(isDatasourceInViewMode);
 
-  if (!allowedToRender || !canShowRamp) return null;
```
Comment (ankitakinger) [inline app/client/src/ce/components/SwitchEnvironment/index.tsx:97]: I think there are same changes on `appsmith-ee/app/client/src/ee/components` folder. Could you check? Do we have a PR to handle the same on EE?

Comment (ayushpahwa) [reply app/client/src/ce/components/SwitchEnvironment/index.tsx:97]: Yeah, the PR is [here](https://github.com/appsmithorg/appsmith-ee/pull/2114)

**Review by ankitakinger — APPROVED** [review]

**Review by ankitakinger — DISMISSED** [review]
LGTM

**Comment by btsgh** [comment]
1. Issue seen - 
https://github.com/appsmithorg/appsmith/assets/101863839/ca55fbeb-37e7-4ef7-8394-8690b224c768

**Review by ChandanBalajiBP — APPROVED** [review]
LGTM

**Pull request merged**